### DISCUSSION TITLE
Change up and down arrows to arrow.u and arrow.d?

### DISCUSSION THREAD

**Discussion by b-vanstraaten** [typst/typst#4177] [Ideas]
Currently, the left and right arrows are accessed with the commands "arrow.l" and "arrow.r". 

However, up and down arrows are accessed through the commands "arrow.t" and "arrow.b" which I guess refers to top and bottom. 

I propose changing the commands to "arrow.u" and "arrow.d" for consistency in name and with latex (https://garsia.math.yorku.ca/MPWP/LATEXmath/node9.html)

**Comment by Enivex** [comment]
This makes a lot of sense in my opinion. It's also relevant for the harpoon symbols, and perhaps more.

**Reply by laurmaedje** [reply]
The `l`, `t`, `b`, `r` convention is used across all of the symbols. Changing that would bring a lot of churn.

**Reply by Enivex** [reply]
> The `l`, `t`, `b`, `r` convention is used across all of the symbols. Changing that would bring a lot of churn.

It makes perfect sense for many things, but arrows have a direction. An arrow points up or down, not top or bottom.

**Reply by b-vanstraaten** [reply]
Perhaps we could alias arrow.t with arrow.p so that either would produce the symbol. Likewise with arrow.b and arrow.d?

**Reply by hz-xiaxz** [reply]
> Perhaps we could alias arrow.t with arrow.p so that either would produce the symbol. Likewise with arrow.b and arrow.d?

I'm with it.
